storage.go

```diff
@@ -5,11 +5,13 @@ import "context"
 // Storage is a interface to store ipam objects.
 type Storage interface {
 	Name() string
-	CreatePrefix(ctx context.Context, prefix Prefix) (Prefix, error)
-	ReadPrefix(ctx context.Context, prefix string) (Prefix, error)
-	DeleteAllPrefixes(ctx context.Context) error
-	ReadAllPrefixes(ctx context.Context) (Prefixes, error)
-	ReadAllPrefixCidrs(ctx context.Context) ([]string, error)
-	UpdatePrefix(ctx context.Context, prefix Prefix) (Prefix, error)
-	DeletePrefix(ctx context.Context, prefix Prefix) (Prefix, error)
+	CreatePrefix(ctx context.Context, prefix Prefix, namespace string) (Prefix, error)
```

Comment:
Should namespace be an optional param here so that it's a backwards compaitable change for people using the library directly?

Reply:
Or have a default in case it's empty?

Reply:
leaving as in the current state is fine from my perspective, if the namespace is an empty string, behavior is as before.